onsite-group-location.html

```diff
@@ -5,116 +5,77 @@
+        src="{{ page.location.image.url | imgix: site.imgix }}?{{
+          site.imgix_params.placeholder_sixteen_nine
+        }}"
+        sizes="{{ site.image_sizes.one_fourth }}"
+        style="width: 100%"
+        data-optimize-img
+      />
     </div>
     <div class="col-md-9">
-      <h3 class="font-family-condensed-extra flush-ends mobile-soft-half-top text-uppercase">Mason Onsite Groups</h3>
-      <a href="#">See what else is happening at Crossroads Mason</a>
-      <p class="font-size-base push-bottom push-half-top">We're all about groups here and want you to have great
-        friends. Below are groups we have
-        inside our building. There are tons of other groups that meet outside the building (trivia night, running,
-        basketball) and you can find those by visiting the <a href="#">Group Tool</a>.</p>
-      <hr>
-    </div>
-  </div>
-  <div class="row push-bottom onsite-group-negative-margin-bottom">
-    <div class="col-md-3">
-      <h4 class="font-family-condensed-extra text-uppercase">The Chaser</h4>
-      <p class="text-gray flush-bottom small"><strong>Perfect for:</strong> Everyone. Post-Weekend Convo Starters</p>
-      <p class="flush-bottom push-quarter-top small"><strong>Monday 6:30-8:00 PM</strong></p>
-      <button class="btn btn-cyan push-top" type="button">Register</button>
-    </div>
-    <div class="col-md-3">
-      <h4 class="font-family-condensed-extra text-uppercase">Mens (AM)</h4>
-      <p class="text-gray flush-bottom small"><strong>Perfect for:</strong> Men of all ages</p>
-      <p class="flush-bottom push-quarter-top small"><strong>Friday 7:00-8:00 AM</strong></p>
-      <button class="btn btn-cyan push-top" type="button">Register</button>
-    </div>
-    <div class="col-md-3">
-      <h4 class="font-family-condensed-extra text-uppercase">Women (PM)</h4>
-      <p class="text-gray flush-bottom small"><strong>Perfect for:</strong> Women of all ages</p>
-      <p class="flush-bottom push-quarter-top small"><strong>Thursday 6:30-8:30 PM</strong></p>
-      <button class="btn btn-cyan push-top" type="button">Register</button>
-    </div>
-    <div class="col-md-3">
-      <h4 class="font-family-condensed-extra text-uppercase">Womens PM (Divorce Care)</h4>
-      <p class="text-gray flush-bottom small"><strong>Perfect for:</strong> Women seeking healing after divorce</p>
-      <p class="flush-bottom push-quarter-top small"><strong>Wednesday 6:30-8:30 PM</strong></p>
-      <button class="btn btn-cyan push-top" type="button">Register</button>
+      <h3
+        class="font-family-condensed-extra flush-ends mobile-soft-half-top text-uppercase"
+      >
+        {{ page.location.name }} Onsite Groups
+      </h3>
+      <a href="#"
+        >See what else is happening at Crossroads {{ page.location.name }}</a
+      >
+      <p class="font-size-base push-bottom push-half-top">
+        {{ page.location.onsite_group_description | markdownify }}
```

Comment:
Wouldn't using `markdownify` inside a paragraph cause invalid markup? You'll end up nesting paragraphs.